ci/use_conda_packages_from_prs.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright (c) 2025, NVIDIA CORPORATION.
+
+# download CI artifacts
+LIBRMM_CHANNEL=$(rapids-get-pr-conda-artifact rmm 1858 cpp)
+RMM_CHANNEL=$(rapids-get-pr-conda-artifact rmm 1858 python)
+
+# make sure they can be found locally
+conda config --system --add channels "${LIBRMM_CHANNEL}"
+conda config --system --add channels "${RMM_CHANNEL}"
+RATTLER_CHANNELS=("-c ${RMM_CHANNEL}" "-c ${LIBRMM_CHANNEL}" "${RATTLER_CHANNELS[@]}")
```

Comment:
Looks like these channels are being treated as `anaconda.org` channels based [on this CI error]( https://github.com/rapidsai/cudf/actions/runs/13827303391/job/38684615505?pr=18260#step:9:543 ):

```
could not find subdir 'noarch' in channel 'https://conda.anaconda.org/tmp/tmp.wPxapVYwTt/'
```

Think adding a `file://` prefix to the channel paths will help

```suggestion
RATTLER_CHANNELS=("-c file://${RMM_CHANNEL}" "-c file://${LIBRMM_CHANNEL}" "${RATTLER_CHANNELS[@]}")
```